test/images-rekt.yaml

```diff
@@ -1 +1,2 @@
 knative.dev/reconciler-test/cmd/eventshub: registry.ci.openshift.org/openshift/knative-eventing-test-eventshub:knative-nightly
```

Comment:
Is it right that you still use knative-nightly here? Should that be v1.10?
I think you said nightly cannot be used anymore. Just curious.

Reply:
Not sure, I think my comments were only related to the code deps from main branch because of a conflict with the changes to Adressable upstream. We should be fine with that image (I think..., let's see).

Reply:
OK. Anyway, the backports helped for multiarch builds here: https://github.com/openshift-knative/serverless-operator/commit/3863526f2d8f5271f275806e100208cde5908101
We should probably use this image as well here so that we have the multiarch image from the beginning and no need to patch later like in the commit above.
Also, the version will change with each release so we probably need to create a template for this file and generate the resulting file automatically. But this can be done in a follow-up. I'd suggest using the quay image in this file for both eventshub and heartbeats. The latest available heartbeats image at quay is v1.9 though: quay.io/openshift-knative/eventing/heartbeats:v1.9 That's because the image was moved from test images to core images in 1.10 and the github action is not building it (this needs a fix!)

Reply:
Created https://issues.redhat.com/browse/SRVCOM-2649